python/lsst/meas/astrom/pessimisticPatternMatcherUtils.cc

```diff
@@ -36,11 +36,20 @@ namespace lsst {
 namespace meas {
 namespace astrom {
 
+namespace {
+
 PYBIND11_MODULE(pessimisticPatternMatcherUtils, mod) {
+    mod.def("find_candidate_reference_pair_range", &find_candidate_reference_pair_range,
+            "Find the candidate pairs between plus/minus the match tolerance.");
+    mod.def("create_pattern_spokes", &create_pattern_spokes,
+            "Create the individual spokes that make up the pattern now that the shift and rotation are "
+            "within tolerance.");
     mod.def("check_spoke", &check_spoke,
             "Test the opening angle between the spokes of our source pattern against the reference.");
```

Comment:
So this would basically be the same (as in keeping "create_pattern_spokes"`&create_pattern_spokes`) but then add after than `"src_ctr"_a`, "src_delta_array"_a correct?

Reply:
Yes, they're all just extra arguments to `mod.def`, nothing else changes.

Reply:
Hmm... after doing this, I get an error during compilation:
`python/lsst/meas/astrom/pessimisticPatternMatcherUtils.cc: In function 'void lsst::meas::astrom::pybind11_init_pessimisticPatternMatcherUtils(pybind11::module_&)':
python/lsst/meas/astrom/pessimisticPatternMatcherUtils.cc:40:90: error: unable to find string literal operator 'operator""_a' with 'const char [9]', 'long unsigned int' arguments
   40 |     mod.def("find_candidate_reference_pair_range", &find_candidate_reference_pair_range, "src_dist"_a,`

Reply:
Ah, sorry, you also need `using namespace pybind11::literals;` at the top of the file, immediately after `namespace py = pybind11;`. This is standard pybind11 boilerplate, so I didn't think to check.